server/app/app.go

```diff
@@ -0,0 +1,87 @@
+package app
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"github.com/haxxorsid/referralboard-private/server/handler"
+	"github.com/haxxorsid/referralboard-private/server/config"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"github.com/gorilla/mux"
+)
+
+// App has router and db instances
+type App struct {
+	Router *mux.Router
+	DB     *gorm.DB
+}
+
+// App initialize with predefined configuration
+func (a *App) Initialize(config *config.Config) {
+	dbURI := fmt.Sprintf("host=%s user=%s password=%s dbname=%s port=%s sslmode=%s TimeZone=%s",
+		config.DB.Host,
+		config.DB.User,
+		config.DB.Password,
+		config.DB.Dbname,
+		config.DB.Port,
+		config.DB.Sslmode,
+		config.DB.TimeZone)
+
+		database, err := gorm.Open(postgres.Open(dbURI), &gorm.Config{})
```

Comment:
We are creating tables in the "app" schema and not using the default schema. So, specify the "app" schema, otherwise, this will fail.

Reply:
added schema name.